### PR TITLE
Tribute page project to be reviewed

### PR DESCRIPTION
**Pull request by AbigiyaTY**
**Tribute page project** 
 
In this project

- Build the homepage in Html
- Add styling to the homepage using CSS and bootstrap

### CODE REVIEW

File: tribute-page/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
+    <!-- Custom Styles -->
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+</head>
+
+<body>
+    <div class="py-4">
+        <h1 class="header1 text-center mt-5"> <strong>Dr. Norman Borlaug</strong> </h1>
+        <p class="text-center">The man who saved a billion lives</p>
+    </div>
```
Comment (AbigiyaTY) [inline tribute-page/index.html:25]: I think putting this in the header tag instead of div is better.

File: tribute-page/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            </br>
+            <li> <strong>2009</strong>- dies at the age of 95.</li>
+            </br>
+        </ul>
+    </div>
+    <div>
+        <p class="quote">
+            <i>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress." <br /> <br /> -- Indian Prime Minister Manmohan Singh</i>
+        </p>
+    </div>
```
Comment (AbigiyaTY) [inline tribute-page/index.html:105]: I think putting this in the section tag instead of div is better.

File: tribute-page/index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    <div>
+        <h5 class="text-center pt-5 pb-2">
+            <strong>If you have time, you should read more about this incredible human being on his<a href="https://en.wikipedia.org/wiki/Norman_Borlaug"> Wikipedia entry.</a></strong>
+        </h5>
+    </div>
```
Comment (AbigiyaTY) [inline tribute-page/index.html:110]: I think putting this in the footer tag instead of div is better.

File: tribute-page/index.html (hunk 1 above)
Comment (AbigiyaTY) [inline tribute-page/index.html:25]: I think putting this in the header tag instead of div is better.

File: tribute-page/index.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>tribute-page.freecodecamp.rocks</title>
+
+    <!-- Link to Montserrat font from Google Fonts-->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:200,200i,300,300i,400,600,600i,700,700i,800,800i,900,900i&display=swap" rel="stylesheet">
+
```
Comment (AbigiyaTY) [inline tribute-page/index.html:13]: I see you didn't you this font so I believe it is better to remove it.

File: tribute-page/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+body {
+    background-color: #eee;
+}
+
+.image {
+    background-color: white;
+    max-width: 100%;
+}
+
+.image img {
```
Comment (AbigiyaTY) [inline tribute-page/css/styles.css:14]: I think it is better just to call it in its img tag since you are only using one image.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @AbigiyaTY, 

Good job so far, this was a good code review!

However, there is something worth pointing out before you move on to your next project.



## Required Changes ♻️

- [ ] Please remember to explain why you're requesting a change, 🙂 for example in your first comment, you mentioned that it's better to use the `section` tag than use `div` tag, but you didn't mention why it's better to do so, for this case, you can say that the `section` tag helps with the semantics of the webpage as it improves its structure, and you can even provide a link if you want to help the student more. 😁

## Optional suggestions

- [ ] Don't be afraid to use emojis as they make the code review more human-friendly 😉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbigiyaTY — COMMENTED** [review]
Hey Abigiya😁

you have done a good job on this project.👏👏👏

#There are changes that are required please check the comments below and make all the required changes one additional thing use linters to check both your HTML and CSS files so that you will have a clean project. Summit for review when you are done.

File: tribute-page/index.html (hunk 4 above)
Comment (Ismailco) [reply tribute-page/index.html:13]: - [ ] [OPTIONAL] You did a good job adding the request changes, please include an explanation for all your changes

**Review by Ismailco — APPROVED** [review]
### Hi @AbigiyaTY, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**